core/PRP/Avatar/plArmatureMod.h

```diff
@@ -79,6 +88,33 @@ class HSPLASMA_EXPORT plArmatureMod : public plArmatureModBase
 protected:
     void IPrcWrite(pfPrcHelper* prc) HS_OVERRIDE;
     void IPrcParse(const pfPrcTag* tag, plResManager* mgr) HS_OVERRIDE;
+
+public:
+    plKey getDefaultMesh() const { return fDefaultMesh; }
+    ST::string getRootName() const { return fRootName; }
```

Comment:
I would probably return strings as references, since they will potentially invoke a deep copy otherwise

Reply:
Makes sense. Most existing getters return `ST::string` by copy though (which is why I did the same thing initially).